src/components/PathwaysList/PathwaysTable.tsx

```diff
@@ -14,69 +14,92 @@ import {
 
 import { usePathwayContext } from 'components/PathwayProvider';
 import { downloadPathway } from 'utils/builder';
+import PathwayModal from './PathwayModal';
 
 import useStyles from './styles';
+import { Pathway } from 'pathways-model';
 
 const PathwaysTable: FC = () => {
   const styles = useStyles();
   const { pathways, deletePathway } = usePathwayContext();
+  const [open, setOpen] = useState(false);
+  const [editablePathway, setEditablePathway] = useState<Pathway>();
+
+  function openEditPathwayModal(pathway: Pathway): void {
+    setOpen(true);
+    setEditablePathway(pathway);
+  }
+
+  const closeEditPathwayModal = useCallback((): void => {
+    setOpen(false);
+  }, []);
 
   return (
-    <TableContainer className={styles.pathwayList}>
-      <Table aria-label="pathway list">
-        <TableHead>
-          <TableRow>
-            <TableCell>Pathway Name</TableCell>
-            <TableCell>Status</TableCell>
-            <TableCell>Last Updated</TableCell>
-            <TableCell></TableCell>
-          </TableRow>
-        </TableHead>
+    <div>
+      <TableContainer className={styles.pathwayList}>
+        <Table aria-label="pathway list">
+          <TableHead>
+            <TableRow>
+              <TableCell>Pathway Name</TableCell>
+              <TableCell>Status</TableCell>
+              <TableCell>Last Updated</TableCell>
+              <TableCell></TableCell>
+            </TableRow>
+          </TableHead>
 
-        <TableBody>
-          {pathways.map(pathway => (
-            <TableRow key={pathway.id}>
-              <TableCell component="th" scope="row">
-                {pathway.name}
-              </TableCell>
+          <TableBody>
+            {pathways.map(pathway => (
+              <TableRow key={pathway.id}>
+                <TableCell component="th" scope="row">
+                  <Button
```

Comment:
To address the font size and capitalization concerns -- maybe instead of a `Button` with `component={Link}` should this just be a pure `Link` component?

Reply:
This is what I was thinking too. It shouldn't need to be a button